### ISSUE TITLE
Storage implant should drop items on crit / death

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#25524]
Apparently it doesn't and round removing an item is pretty toxic.

**Comment by deltanedas** [comment]
they get dropped when gibbed or implant is removed

**Comment by deltanedas** [comment]
> Maybe drop items as the body starts rotting

makes sense since its supposed to be stored under your skin or something, and if you are rotting you wont care since a borg cant use items, and being valid doesnt affect borgability

**Comment by Ghagliiarghii** [comment]
As Carve said, there are methods to get items out of a storage implant by force, and some RP ways to determine if someone has an implant (examples are if you search them for a nuke disk and it's not in their bag/pockets but the locator points right towards them, if you find the empty implanter somewhere with their dna on the needle, if they are doing other valid things when there's evidence and there are reports they stole an item that's not on them when you search)

The implanter is relatively expensive because it lets you hide these items and I don't like these proposed solutions to what I view as a non-issue. I feel it may also cause worse meta-gaming with intentionally critting/killing and then reviving arrested persons. Gibbing is serious and for that reason it's relatively uncommon as sec and the reverse implanter is just annoying enough to discourage metagaming where it would be an automatic part of an arrest. Perfectly balanced.

**Comment by deepdarkdepths** [comment]
If this is implemented, sec on LRP is just going to gun down anyone they suspect of having an implant.

**Comment by Hmeister-real** [comment]
Maybe storage implants should become visible on someone who has no clothing on when they ar mid-rot (jumpsuits or outserclothing)? like at the "They don't look fresh anymore" theres also "You are able to see bulges under their skin"

**Comment by UbaserB** [comment]
> Maybe storage implants should become visible on someone who has no clothing on when they ar mid-rot (jumpsuits or outserclothing)? like at the "They don't look fresh anymore" theres also "You are able to see bulges under their skin"

this actually sounds like a better idea

**Comment by keronshb** [comment]
It should show on mid-late stage rotting like others said. It's meant to be a stealth item. If it drops on crit or death immediately, the implant itself becomes useless.

Say if they revive you after you died, there goes the item you hid away.

As others said, discovery of a storage implant is meant by deduction.

**Comment by Carmell0** [comment]
If this gets merged we will see a large increase in alleged thieves being beat to death

**Comment by deltanedas** [comment]
issue with gibbing refactor, it used to drop implants when gibbed
